File: SoftInvPersistance/src/main/java/pe/edu/pucp/softinv/dao/ServicioDAO.java
package pe.edu.pucp.softinv.dao;

import pe.edu.pucp.softinv.model.Servicio.ServicioDTO;

import java.util.ArrayList;

public interface ServicioDAO {
    public Integer insertar(ServicioDTO servicio);
    public ServicioDTO obtenerPorId(Integer id);
    public Integer modificar(ServicioDTO servicio);
    public Integer eliminar(ServicioDTO servicio);
    ArrayList<ServicioDTO> listarServiciosDeEmpleado(Integer empleadoID);
}

File: C#/SoftInvPersistance/DAO/ServicioDAO.cs
using softinv.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.DAO
{
    interface ServicioDAO
    {
        int insertar(ServicioDTO servicio);
        ServicioDTO obtenerPorId(int id);
        int modificar(ServicioDTO servicio);
        int eliminar(ServicioDTO servicio);
        IList<EmpleadoDTO> listarEmpleadosDeServicio(int servicioID);
    }
}
